pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -57,34 +55,29 @@ def __init__(
             )
 
     def run(
-        self, job_params: list[JobParams] | None = None
+        self, job_params: list[JobParams] = []
```

Comment:
You're right. It's not something I usually do and I tend to never change the arguments in place, but I agree it's probably best to use `None` just in case.

Reply:
I changed all the mutable defaults to avoid these bugs